### PR TITLE
[Publishing] remove GridThemeProvider

### PR DESCRIPTION
**Pull request by eessex**
Related slack convo: https://artsy.slack.com/archives/C2TQ4PT8R/p1550601288097000

### CODE REVIEW

File: src/Components/Publishing/Article.tsx [hunk 1]
```diff
@@ -107,20 +106,15 @@ export class Article extends React.Component<ArticleProps> {
     return (
       <MediaContextProvider>
         <Theme>
-          <GridThemeProvider gridTheme={themeProps.grid}>
```
Comment (damassi) [inline src/Components/Publishing/Article.tsx]: For ref this was moved into https://github.com/artsy/palette/blob/master/src/Theme.tsx#L330

**Comment by eessex** [comment]
Pushed this up with no-verify because I was getting unrelated errors from the `ArtworkGrid` in master, but looks like they will need to be addressed for this to be merged.

**Comment by damassi** [comment]
Just noticed that with https://github.com/artsy/reaction/pull/2037 as well. @ds300 is on it.

**Comment by mzikherman** [comment]
If this gets rebased against master should be 💚

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 37bf85b5da998f076b7984cacec8191eb507c49d

https://deploy-preview-2038--artsy-reaction.netlify.com

**Comment by artsyit** [comment]
:rocket: PR was released in v14.0.4 :rocket:

**Pull request merged**